azure-pipelines.yml

```diff
@@ -1,19 +1,161 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yy)$(DayOfYear).$(BuildCounter)
 
 trigger:
-- main
+  - main
+  - release/*
 
-pool:
-  vmImage: ubuntu-latest
+pool: $(WindowsPool)
 
-steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+variables:
+  - name: BuildConfiguration
+    value: "release"
+  - name: BuildPlatform
+    value: "any cpu"
+  - name: BuildCounter
+    value: $[counter(format('{0:yyyyMMdd}', pipeline.startTime), 1)]
+  - name: WindowsPool
+    value: "NautilusBuild"
 
-- script: |
-    echo Add other tasks to build, test, and deploy your project.
-    echo See https://aka.ms/yaml
-  displayName: 'Run a multi-line script'
+stages:
+
+  - stage: PERFORM_DEPENDENCYCHECK_DOTNETBUILD_DOTNETTEST_AND_PUBLISH
+    displayName: "Build (DependencyChecker, Dotnet Build & Restore , Dotnet Test and Publish Artifact )"
+    dependsOn: []
+    jobs:
+      - job: Dependencychecker
+        workspace:
+          clean: all
+        displayName: "Dependencychecker"
+        steps:
+          - task: UseDotNet@2
+            displayName: 'Use .NET 6.0.x sdk'
+            inputs:
+              packageType: sdk
+              useGlobalJson: true
+              workingDirectory: '$(Build.SourcesDirectory)\src'
+
+          - task: DotNetCoreCLI@2
+            displayName: ".Net Core - NuGet restore test projects only"
```

Comment:
This should be restore non-test projects only

Reply:
> This should be restore non-test projects only

Corrcted as mentioned